mon-pix/tests/integration/components/steps_test.js

```diff
@@ -91,6 +91,26 @@ module('Integration | Component | steps', function (hooks) {
     });
 
     module('on the last page', function () {
+      test('should display information', async function (assert) {
+        // given
+        // when
+        const screen = await render(hbs`<CertificationInstructions::Steps/>`);
+        for (let i = 0; i < 4; i++) {
+          await click(screen.getByRole('button', { name: "Continuer vers l'écran suivant" }));
+        }
+
+        // then
+        assert.dom(screen.getByRole('heading', { name: 'Règles à respecter', level: 2 })).exists();
+        assert.dom(screen.getByText('En certification, il est interdit de :')).exists();
+        assert
+          .dom(
+            screen.getByRole('checkbox', {
+              name: 'En cochant cette case, je reconnais avoir pris connaissances de ces règles et je m’engage à les respecter.',
+            }),
+          )
+          .exists();
```

Comment:
On souhaite tester que le bouton est `enable` sur clic donc on doit aussi tester que sans le clic il est `disable`
```suggestion
          .isDisabled();
```